src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -55,6 +56,8 @@ public class HelpWindow extends UiPart<Stage> {
             + "\n\n"
             + HELP_SHOW_COMMAND
             + "\n\n"
+            + HELP_SORT_COMMAND
```

Comment:
Should probably state that sort is permanent and it sorts by alphabetical order and to unsort, use list command.